common/src/commonMain/kotlin/org/hisp/dhis/common/screens/InputDropDownScreen.kt

```diff
@@ -0,0 +1,102 @@
+package org.hisp.dhis.common.screens
+
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import org.hisp.dhis.mobile.ui.designsystem.component.ColumnComponentContainer
+import org.hisp.dhis.mobile.ui.designsystem.component.InputDropDown
+import org.hisp.dhis.mobile.ui.designsystem.component.InputShellState
+import org.hisp.dhis.mobile.ui.designsystem.component.SubTitle
+import org.hisp.dhis.mobile.ui.designsystem.component.Title
+import org.hisp.dhis.mobile.ui.designsystem.theme.Spacing
+import org.hisp.dhis.mobile.ui.designsystem.theme.TextColor
+
+@Composable
+fun InputDropDownScreen() {
+    ColumnComponentContainer {
+        val options = listOf("Option 1", "Option 2", "Option 3", "Option 4", "Option 5", "Option 6", "Option 7")
+        var expanded by rememberSaveable { mutableStateOf(false) }
+
+        Title("Input Dropdown", textColor = TextColor.OnSurfaceVariant)
+
+        SubTitle("Basic Input Dropdown ", textColor = TextColor.OnSurfaceVariant)
+        var selectedItem by rememberSaveable { mutableStateOf<String?>(null) }
+        InputDropDown(
+            title = "Label",
+            state = InputShellState.UNFOCUSED,
+            selectedItem = selectedItem,
+            onResetButtonClicked = {
+                selectedItem = null
+            },
+            onArrowDropDownButtonClicked = {
+                expanded = !expanded
+            },
+        )
+        DropdownMenu(
```

Comment:
The way the `DropdownMenu` is configured right now, it moves the content by px or so when it's opened. When we move the dropdown into the component, let's wrap the content and dropdown inside a box and use dropdown offset to make sure they are overlap.

Reply:
This relates to above comment.

Reply:
Could you wrap up the dropdown menu and input it inside a box? Or else it's pushing the content down a bit every time it's opened in the sample.

Reply:
Will be useful to showcase the proper usage when they are configuring it inside the app.

Reply:
@msasikanth Fixed.